sample/src/androidTest/java/com/schibsted/spain/barista/sample/RadioButtonsReallyFarAwayTest.java

```diff
@@ -62,4 +65,10 @@ public void checkRadioButtonsByPosition_withLastOption() {
     clickRadioButtonPosition(R.id.radiogroup_really_far_away, 1);
     assertDisplayed("" + R.id.last_item_really_far_away);
   }
+
+  @Test
+  public void farAwayCheckRadioButtonContainsText() {
+    clickRadioButtonPosition(R.id.radiogroup_really_far_away, 0);
```

Comment:
If we just need to scroll to this view, I suggest to use `scrollTo` instead of clicking it, because newbies could miss why we need to click it before doing the assertion. But it's a detail :·)